airflow/operators/postgres_to_gcs.py

```diff
@@ -88,7 +90,8 @@ def convert_type(self, value, schema_type):
         Decimals are converted to floats. Times are converted to seconds.
         """
         if isinstance(value, (datetime.datetime, datetime.date)):
-            return time.mktime(value.timetuple())
+            return pendulum.parse(
+                value.isoformat()).in_tz('UTC').float_timestamp
```

Comment:
Why UTC?

Reply:
@RosterIn  At the time I wrote this, I understood that the ISO-8601 string representation for `value` could have a timezone offset and thought extra handling necessary to convert the parsed datetime in the local time.

For a fact, this extra conversion isn't necessary to perform. I'll remove it.